packages/claude-hooks/src/quality-check/sub-agent-config.ts

```
@@ -0,0 +1,459 @@
+        break
+      case '--subagent-sensitivity':
+        if (['conservative', 'balanced', 'aggressive'].includes(nextArg)) {
+          config.escalationSensitivity = nextArg as
+            | 'conservative'
+            | 'balanced'
+            | 'aggressive'
+          i++
+        }
+        break
+      case '--subagent-max-rate':
+        const rate = parseFloat(nextArg)
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)